joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/MavenDependencies.scala

```diff
@@ -47,8 +47,9 @@ object MavenDependencies {
   }
 
   private[dependency] def get(projectDir: Path): Option[collection.Seq[String]] = {
+    val isWindows = System.getProperty("os.name").toLowerCase.startsWith("windows")
```

Comment:
You may want to use `import scala.util.Properties.isWin` and use it like e.g., in https://github.com/joernio/joern/blob/master/semanticcpg/src/main/scala/io/shiftleft/semanticcpg/utils/ExternalCommand.scala#L64

Reply:
yes, what max said